DigitalLearningSolutions.Web/Controllers/LearningMenuController/LearningMenuController.cs

```diff
@@ -110,7 +110,7 @@ public IActionResult Section(int customisationId, int sectionId)
             return View("Section/Section", model);
         }
 
-        [Route("/LearningMenu/1/1/Diagnostic")]
+        [Route("/LearningMenu/18438/993/Diagnostic")]

```

Comment:
I meant actually pass in values to this method, don't hardcode them. There's no reason to hardcode them as we should have this information available on the section page, right? 🙂

Reply:
`SectionID` doesn't seem to be available on the section page yet, Shiv is adding that now. I can change it so that it takes parameters but hardcode those values on the section page for now?

Reply:
Okay. @SChandarana the section id is available in the controller action so it's just a question of passing that from the action to the view model 🙂